build.assets/Dockerfile-multiarch

```diff
@@ -25,12 +27,23 @@ RUN echo "armhfp" > /etc/yum/vars/basearch && \
 ## ARM 64 #####################################################################
 FROM base AS platform-setup-arm64
 ARG GOLANG_ARCH=arm64
+ARG RUST_ARCH=aarch64-unknown-linux-gnu
 
 # Installing the kerenel packages causes the update to hang on aarch64, so we
 # skip upgrading them.
 ARG YUM_UPDATE_FLAGS="--exclude kernel-*"
 RUN echo "aarch64-redhat-linux-gnu" > /etc/rpm/platform
 
+## 386 ########################################################################
+FROM base AS platform-setup-386
+ARG GOLANG_ARCH=386
+ARG RUST_ARCH=i686-unknown-linux-gnu
+
+## AMD 64 #####################################################################
+FROM base AS platform-setup-amd64
+ARG GOLANG_ARCH=amd64
+ARG RUST_ARCH=x86_64-unknown-linux-gnu
```

Comment:
Why do we need 386/AMD64 here? We're using different docker images for that.

Reply:
That was the original idea for this specific Dockerfile: to use a single build image to build for all platforms. I guess, I am a little late replying you here, and we've already discussed it in DM. Just to Drone builds minimal images, so, let me leave and use it here for Drone and we'll sync about again later.